### PR TITLE
Add links to development snapshot docs.

### PR DESCRIPTION
**Pull request by jchodera**
This adds links to the development snapshot docs to the bottom of the "Documentation" page.

### CODE REVIEW

File: landing-page/documentation.html [hunk 1]
```diff
@@ -93,6 +94,47 @@ <h2 class="page-header">Workshop Videos</h2>
...
+              <h3>
+                <a href="http://docs.openmm.org/development/developerguide/index.html">Dev Guide</a>
+              </h3>
+              <p class="enlarge">
+                This section provides in-depth information on the code-base and how to contribute to the development of OpenMM itself.
+              </p>
+            </div>
+            <div class="col-md-3 column">
+              <h3>
+                <a href="http://docs.openmm.org/development/api-python/annotated.html">Python API
```
Comment (rmcgibbo) [inline landing-page/documentation.html]: no, don't link here. annotated.html is from the old doxygen docs.

File: landing-page/documentation.html [hunk 2]
```diff
@@ -93,6 +94,47 @@ <h2 class="page-header">Workshop Videos</h2>
...
+                </a>
+              </h3>
+              <p class="enlarge">
+                The Python API provides information about the classes and methods available to Python developers.
+              </p>
+            </div>
+            <div class="col-md-3 column">
+              <h3>
+                <a href="http://docs.openmm.org/development/api-c++/namespaceOpenMM.html">C++ API
```
Comment (rmcgibbo) [inline landing-page/documentation.html]: same. just use the index page.

**Comment by rmcgibbo** [comment]
maybe you could add a google analytics tracker or something too?

**Comment by jchodera** [comment]
Whoops!  Thanks!

**Comment by jchodera** [comment]
> maybe you could add a google analytics tracker or something too?

Good idea!  Is there already one set up?

**Comment by jchodera** [comment]
> maybe you could add a google analytics tracker or something too?

Maybe this is a job for @mpiercy in a separate PR?  I imagine you guys might want to control the google analytics statistics page.

**Comment by jchodera** [comment]
Or we could use the [one from your openmm-webbuilder](https://github.com/rmcgibbo/openmm-webbuilder/blob/master/public/index.html#L39-L50)?

**Comment by jchodera** [comment]
OK, added google analytics tracking, just for the hell of it.

Oddly enough, I also seem to have the tracking data for http://folding.stanford.edu in my analytics dashboard.  I am really not sure how that happened.

**Comment by rmcgibbo** [comment]
lgtm.

**Comment by jchodera** [comment]
Not sure who can merge this.  @mpiercy?

**Pull request merged**